docs/pages/desktop-access/getting-started.mdx

```diff
@@ -314,6 +314,62 @@ Computer Configuration > Policies > Administrative Templates > Windows Component
   ![Disable Require](../../img/desktop-access/disable.png)
 </Figure>
 
+### Configure certificate for RDP connections
+
+<Admonition type="note" title="Secure Cipher Suites">
+  Teleport's RDP client supports only secure algorithms
+  for making TLS connections, so we have to configure a Windows server
```

Comment:
Is "a Windows server" one that we've referred to earlier in this guide? Is it the one where we've been configuring our service account and GPOs? We might want to specify which host commands in this section will be run on.

Reply:
Yes this is the same Windows server. I'll add a clarification.